coding_systems/base/coding_system_base.py

```diff
@@ -42,6 +42,8 @@ class BaseCodingSystem(ABC):
     # are not guaranteed to be exhaustive in future.
     csv_headers = {"code": ["code"], "term": ["term"]}
 
+    is_builder_compatible = False
+
     def __init__(self, database_alias):
         self.database_alias = database_alias
```

Comment:
I wonder if something like this declared on `BaseCodingSystem` would be better:

```python
    @property
    def is_builder_compatible(self):
        return isinstance(self, BuilderCompatibleCodingSystem)
```
I appreciate it could still be overridden just as easily on any coding system class, but it keeps the logic in one place, and I think makes it clearer what makes something builder compatible.

Reply:
Oh I like this idea

Reply:
turns out there's no such thing as a class property, so had to use a `@classmethod` instead